android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotificationListenerService.java

```diff
@@ -7,6 +7,11 @@
 import android.os.Handler;
 import android.os.Looper;
 import android.util.Log;
+import android.os.Build.VERSION;
+import android.app.usage.UsageStatsManager;
+import android.app.usage.UsageStats;
+import android.content.Context;
+
```

Comment:
I don't think any of the extra imports are also required except `import android.os.Build.VERSION;`.